org.eclipse.jdt.ui/ui refactoring/org/eclipse/jdt/internal/ui/refactoring/ExtractTempWizard.java

```diff
@@ -102,8 +104,11 @@ public void createControl(Composite parent) {
 			layouter.perform(label, text, 1);
 
 			addReplaceAllCheckbox(result, layouter);
-			addDeclareFinalCheckbox(result, layouter);
+			addReplaceAllInFileCheckbox(result, layouter);
```

Comment:
After testing, this checkbox should be indented and should be a sub-option of replace all (i.e. is disabled if replace all is not checked).  I tried setting the new option without the previous one and it changed one instance in each method which doesn't make sense.